src/ptc/util/jms.clj

```diff
@@ -30,7 +30,6 @@
    ::copy   true     :analysis_version_number     :analysisCloudVersion
    ::copy   true     :chip_well_barcode           :chipWellBarcode
    ::copy   true     :reported_gender             :gender
-   ::copy   true     :sample_alias                :sampleAlias
```

Comment:
I think `sample_alias` is a required parameter in the wfl. It just so happens that the `collaboratorSampleId` (which is not included in the message) is the `sampleAlias` and required in params.txt. Maybe we need both?